### ISSUE TITLE
No XML Documentation for core dotnet assemblies in VS Code

### ISSUE THREAD

**Issue by mgwelch** [dotnet/runtime#39927]
# Issue Title

A brand new (re)-install of dotnet core (dotnet-sdk-3.1.302-osx-x64.pkg) on macOS does not appear to have xml documentation for all assemblies. (At least VS Code doesn't find it. However, as I write this I see that Visual Studio for Mac is giving me documentation pop-ups)

# General

I'm running macOS Catalina (10.15.6) and VS Code (1.47.2). When I use IntelliSense or hover over a system method I do not get any pop help. I asked a [question](https://stackoverflow.com/questions/63044879/how-do-i-install-dotnet-framework-help-files-for-vs-code) on stack overflow to get assistance. One answer directed me where to look for the XML files. I was missing xml files for mscorlib and System.IO in the location he suggested. However a broader look for the xml does find these files in NugetFallbackFolder. It's not clear if this is a VS Code issue (not looking in correct location) or a dotnet core issue (not supplying the core documentation in the core location. 

Doing a search as the responder suggested:

```bash
➜  share find /usr/local/share/dotnet -iname 'System.IO.dll' | grep 'NETCore.App' | grep -v Access
/usr/local/share/dotnet/shared/Microsoft.NETCore.App/3.1.5/System.IO.dll
/usr/local/share/dotnet/shared/Microsoft.NETCore.App/2.2.5/System.IO.dll
/usr/local/share/dotnet/shared/Microsoft.NETCore.App/2.1.8/System.IO.dll
/usr/local/share/dotnet/shared/Microsoft.NETCore.App/3.1.6/System.IO.dll
/usr/local/share/dotnet/shared/Microsoft.NETCore.App/2.1.9/System.IO.dll
/usr/local/share/dotnet/shared/Microsoft.NETCore.App/2.1.19/System.IO.dll
/usr/local/share/dotnet/shared/Microsoft.NETCore.App/2.1.20/System.IO.dll
/usr/local/share/dotnet/packs/Microsoft.NETCore.App.Ref/3.1.0/ref/netcoreapp3.1/System.IO.dll
➜  share find /usr/local/share/dotnet -iname 'System.IO.xml' | grep 'NETCore.App' | grep -v Access
```

But expanding the search: 

```
find /usr/local/share/dotnet -iname 'System.IO.xml' | grep -v Access
// lots of results not shown 
/usr/local/share/dotnet/sdk/NuGetFallbackFolder/system.io/4.1.0/ref/netstandard1.5/fr/System.IO.xml
/usr/local/share/dotnet/sdk/NuGetFallbackFolder/system.io/4.1.0/ref/netstandard1.5/System.IO.xml
/usr/local/share/dotnet/sdk/NuGetFallbackFolder/system.io/4.1.0/ref/netstandard1.5/es/System.IO.xml
```

Something similar happens for mscorlib

The end result is that in VS Code there is no help for core APIs.

**Comment by mgwelch** [comment]
Ok, I have more data on this issue. I had opened the issue on vscode, but today I discovered that in Visual Studio for Mac I also wasn't seeing help for some methods.

tl;dr. The problem seemed to be if the project was netstandard2.0. Upgrading it to netstandard2.1 resolved the issue. (Could have sworn I saw the issue with console project as well but haven't tried to investigate further).

See the rest: https://github.com/microsoft/vscode/issues/103289#issuecomment-663991952

**Comment by danmosemsft** [comment]
Not sure who might have context here - maybe @safern

**Comment by saucecontrol** [comment]
This would seem to be another instance of https://github.com/dotnet/standard/issues/1527

**Comment by safern** [comment]
Yeah I agree with @saucecontrol if help is working for netstandard2.1 and netcoreapp3.1 projects this is definitely an issue with the .NET Standard 2.0 ref pack not containing the correct .xml file in it.

**Comment by ericstj** [comment]
System.IO.dll is a full facade on .NETCore, which is why you didn't see its XML file in the refpack.  All types are forwarded to System.Runtime and System.Runtime.Extensions.  For which you will see xml files for in `packs/Microsoft.NETCore.App.Ref/3.1.0/ref/netcoreapp3.1`

Agreed this is a duplicate of dotnet/standard#1527.
